pom.xml

```diff
@@ -141,6 +141,13 @@
 				<scope>provided</scope>
 			</dependency>
 
+			<dependency>
+				<groupId>com.thoughtworks.xstream</groupId>
+				<artifactId>xstream</artifactId>
+				<version>1.4.20</version>
```

Comment:
Why did you add this dependency? And what scope does it have?

Reply:
I got errors like `NoClassDefFoundError: Could not initialize class com.thoughtworks.xstream.converters.collections.TreeMapConverter` before so I added this line. I think those issues got resolved by my other changes so we don't need this now. I updated the PR by removing this.